src/components/Compare/Compare.style.tsx

```diff
@@ -243,77 +243,57 @@ export const TableHeadContainer = styled(TableHead)<{ isModal?: Boolean }>`
 export const Population = styled.span`
```

Comment:
Looks like there's a slight placement issue with the slider on mobile...  (Nearby is falling off)

![image](https://user-images.githubusercontent.com/206364/92639096-319ff380-f290-11ea-9cd4-b9c15494a2b5.png)